README.md

```diff
@@ -93,6 +97,20 @@ The basic operation of the tool is this:
 The data can then be loaded by the ilib-localedata package for use in the various ilib-*
 packages.
 
+## Basic Operation for legacy version of ilib
+1. Write the `inc` file to specify which files to assemble.
+   If not, the default file that the tool (`./src/ilib-ilib-assemble-inc.js`)has will be used.
+2. Read the javscript files from the inc file list and extract the list of dependent JS and locale data.
+3. After the tool has been executed, the two style files will be placed the `output-dir`.
```

Comment:
"has been executed" -> sounds like you have sentenced the tool to death!  ;-)
use "has finished executing" instead.

"two style files" -> "two types of files (code, data)"

Reply:
oh! ^^;; I updated it. Thank you!